kickstarts/centos8_build_machine.ks

```diff
@@ -21,11 +21,11 @@ reboot
 graphical
 
 # Use network installation
-url --url="http://mirror.centos.org/centos/8/BaseOS/x86_64/os/"
-repo --name="AppStream" --baseurl=http://mirror.centos.org/centos/8/AppStream/x86_64/os/
-repo --name="PowerTools" --baseurl=http://mirror.centos.org/centos/8/PowerTools/x86_64/os/
-repo --name="extras" --baseurl=http://mirror.centos.org/centos/8/extras/x86_64/os/
-repo --name="epel" --baseurl=https://mirror.atl.genesisadaptive.com/epel//8/Everything/x86_64/
+url --url="http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/"
+repo --name="AppStream" --baseurl=http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/
+repo --name="PowerTools" --baseurl=http://mirror.centos.org/centos/8-stream/PowerTools/x86_64/os/
+repo --name="extras" --baseurl=http://mirror.centos.org/centos/8-stream/extras/x86_64/os/
+repo --name="epel" --baseurl=https://mirror.atl.genesisadaptive.com/epel/8/Everything/x86_64/
```

Comment:
I also noticed this used a specific mirror instead of the generic mirror or --mirrorlist (do you know why?)

Reply:
```suggestion
repo --name="epel" --baseurl=https://download.fedoraproject.org/pub/epel/8/Everything/x86_64/
```